development.bats

```diff
@@ -25,30 +25,6 @@ testCommit() {
     assert_failure
 }
 
-@test "check_repo fails if repo precommit is missing gitleaks" {
```

Comment:
If our understanding that local pre-commit hooks don't override global pre-commit hooks is correct, then these tests are superfluous. We have added a unit test to prove that local pre-commit hooks and the global hooks for gitleaks run simultaneously

Reply:
With appropriate unit tests added that should be sufficient to mitigate worries around this.